packages/async-ssr-manager/src/index.ts

```diff
@@ -7,18 +7,55 @@ import {AsyncSsrManager} from './internal/async-ssr-manager';
 import {validateConfig} from './internal/validate-config';
 
 export interface AsyncSsrManagerConfig {
+  /**
+   * Render timeout in milliseconds.
+   */
   readonly timeout?: number;
 }
 
+/**
+ * The Async SSR Manager enables the integrator to render any given composition
+ * of React Feature Apps in multiple render passes until all Feature Apps and
+ * Feature Services have finished their asynchronous operations.
+ *
+ * The integrator should provide the Async SSR Manager only on the server.
+ *
+ * Feature Apps and Feature Services should declare the Async SSR Manager as an
+ * optional dependency. Its presence can be used to determine whether they are
+ * currently rendered on the server or on the client.
+ */
 export interface AsyncSsrManagerV0 {
+  /**
+   * This method is intended for the integrator. It calls the given render
+   * funtion in a loop as long as there are unresolved rerender promises, and
```

Comment:
`function`

Reply:
`It calls the given render function in a loop every time all rerender promises are resolved, until a render pass doesn't yield new rerender promises.`